packages/server/lib/plugins/index.js

```diff
@@ -224,13 +62,18 @@ const _setPluginsProcess = (_pluginsProcess) => {
   pluginsProcess = _pluginsProcess
 }
 
+const getPluginIpcHandlers = () => {
+  return handlers
+}
+
 module.exports = {
   getPluginPid,
   execute,
   has,
   init,
```

Comment:
Looks like you're still exporting 'init' even though the body has been removed, causing an error.